code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -89,10 +89,6 @@
 		app.wiz_team = master_wizard.wiz_team
 		master_wizard.wiz_team.add_member(app_mind)
 	app_mind.add_antag_datum(app)
-	//TODO Kill these if possible
-	app_mind.assigned_role = "Apprentice"
```

Comment:
wiz apprentice should be a seperate display role name

Reply:
![image](https://user-images.githubusercontent.com/87972842/216933892-aacc58ae-e923-4268-b76c-70c0a47c36ca.png)